worker/process-image.js

```diff
@@ -103,6 +110,39 @@ function uploadToS3 (s3, path, key, metadata, thumbPath, callback) {
   });
 }
 
+function translateJPEG2000 (ext, path, key, callback) {
+  if (ext !== '.jp2') {
+    return callback(null, path, key);
+  }
+
+  if (!config.gdalTranslateBin) {
+    throw new Error('GDAL bin path missing.');
+  }
+
+  log(['debug'], 'Converting JPEG2000 to TIFF.');
+  var parsedPath = pathTools.parse(path);
+  var outPath = pathTools.join(parsedPath.dir, parsedPath.name) + '.tif';
+
+  var cmd = `${config.gdalTranslateBin} -of GTiff ${path} ${outPath} ` +
```

Comment:
Is there any way that malicious content could be introduced through `path` or `outpath`? If so, consider using the `execFileSync` variant that puts arguments into an array (it prevents spaces being interpreted in a potentially bad bad).